src/components/loading/loading.scss

```diff
@@ -7,7 +7,7 @@
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  color: $tone-3-color;
+  color: var(--playkit-tone-3-color);
```

Comment:
so using the sass variables doesn't work if the values ? so should we even export the sass variables ?

Reply:
My idea that plugins should not use `ui` repo directly, only from kaltura-player. Issue that if we update kaltura-player with new `ui` plugins has still have outdated `ui` dependency in pakage.json.